### PR TITLE
[RA-1 Ch03] 3.2.4. Virtual Networking

### PR DESCRIPTION
**Pull request by msidana**
Added initial content

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter03.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ Images are stored using the OpenStack Glance service discussed below in Section
...
 
 <a name="3.2.4"></a>
-### 3.2.4. Virtual Networking – Neutron standalone 
-Content to be developed
+### 3.2.4. Virtual Networking Neutron standalone 
+Neutron is an OpenStack project that provides "network connectivity as a service" between interface devices(e.g., vNICs) managed by other OpenStack services (e.g., nova). Neutron allows users to create networks, subnets, ports, routers etc and handles traffic isolation between different subnets - within as well as across project(s). Multiple network segments are supported by Neutron via ML2 plugins to simultaneously utilize varierty of layer 2 networking technologies like VLAN, VxLAN, GRE etc. Neutron also allows to create routers to connect layer 2 networks via "neutron-l3-agent". In addition, floating IP support is also provided that allows a project VM to be accessed using a public IP.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.md]: @msidana Great start.
"... handles traffic isolation between different subnets .." -- I don't think neutron directly "handles traffic isolation"; mechanisms for traffic isolation are VLANs and tunneling.

Should some of the limitations and workarounds be mentioned?

Comment (msidana) [reply doc/ref_arch/openstack/chapters/chapter03.md]: Neutron doesn't handles directly but facilitates such isolation by using different type drivers/mechanism drivers like VLANs, VxLANs, GRE (Generic Routing Encapsulation) tunnels etc. For neutron API consumer, this is abstracted and provided by neutron.

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter03.md]: @msidana Can you incorporate into the text? Thanks

Comment (msidana) [reply doc/ref_arch/openstack/chapters/chapter03.md]: Done.

**Comment by pgoyal01** [comment]
Fixes #632

**Comment by msidana** [comment]
@rabi-abdel Pls review and merge.

**Comment by rabi-abdel** [comment]
can we get one vendor to approve this @pgoyal01 please identify someone.

**Comment by pgoyal01** [comment]
Something didn't work as the patch still exists after the merge

**Pull request merged**